### PR TITLE
Review bidders to cover not matched imp.id with bid.impid

### PR DESCRIPTION
**Pull request by rpanchyk**

### CODE REVIEW

File: src/main/java/org/prebid/server/bidder/ix/IxBidder.java [hunk 1]
```diff
@@ -132,8 +132,8 @@ private static Site modifySite(Site site, ExtImpIx extImpIx) {
         return site == null
                 ? null
                 : site.toBuilder()
-                        .publisher(modifyPublisher(site.getPublisher(), extImpIx.getSiteId()))
```
Comment (RodionOrets) [inline src/main/java/org/prebid/server/bidder/ix/IxBidder.java:135]: Indentation removal was accidendal or purposely?

File: src/main/java/org/prebid/server/bidder/synacormedia/SynacormediaBidder.java [hunk 2]
```diff
@@ -150,7 +150,7 @@ private static BidType getBidType(String impId, List<Imp> imps) {
         for (Imp imp : imps) {
             if (imp.getId().equals(impId)) {
                 if (imp.getBanner() != null) {
-                    break;
```
Comment (RodionOrets) [inline src/main/java/org/prebid/server/bidder/synacormedia/SynacormediaBidder.java:153]: Looks much better now

**Review by RodionOrets — APPROVED** [review]

**Comment by RodionOrets** [comment]
Other than Rubicon and Addopler bidder unit tests changes aren't there. Looks like we could add small test to each adapter to cover missing imp NPE situation

File: src/main/java/org/prebid/server/bidder/ix/IxBidder.java (hunk 1 above)
Comment (rpanchyk) [reply src/main/java/org/prebid/server/bidder/ix/IxBidder.java:135]: Seems this is default IDE formatter. Gues we my talk offline on it.

**Pull request merged**